frontend/src/Components/ChatList.js

```diff
@@ -10,9 +10,11 @@ export default function ChatsList({ onSelectChat }) {
   const [openDialog, setOpenDialog] = useState(false); // State to handle dialog visibility
   const [username, setUsername] = useState(""); // State for the input field
 
+  const API_URL = process.env.REACT_APP_HTTPS_API_URL;
+
```

Comment:
it would be better to move this constant into separate file. name it `constants.js`. all other constants in the future should be there